app/config/sonata_media.yml

```diff
@@ -0,0 +1,26 @@
+sonata_media:
+    default_context: default
+    db_driver: doctrine_mongodb
+    contexts:
+        default:
+            providers:
+                - sonata.media.provider.youtube
+                - sonata.media.provider.image
+                - sonata.media.provider.file
+
+            formats:
+                small: { width: 100 , quality: 70}
+                big:   { width: 500 , quality: 70}
+
+    cdn:
+        server:
+            path: /uploads/media
```

Comment:
I create web/uploads/media/.getkeep and add to gitignore 
/web/uploads/*
!/web/uploads/media/.gitkeep
but git gui not see this file,i dont know why